content/overview/getting-started/_index.en.md

```diff
@@ -82,11 +85,11 @@ wash claims inspect wasmcloud.azurecr.io/echo:0.3.2
   None
 ```
 
-To start the HTTP server capability provider, again use the web UI and click **Start Provider** and then select _From Registry_. Supply the OCI URL `wasmcloud.azurecr.io/httpserver:0.14.4` and leave the _link name_ set to `default`. You should now see this capability provider running.
+To start the HTTP server capability provider, again use the web UI and click **Start Provider** and then select _From Registry_. Supply the OCI URL `wasmcloud.azurecr.io/httpserver:0.14.6` and leave the _link name_ set to `default`. You should now see this capability provider running, and after a short time it should report its status as Healthy.

```

Comment:
after a short time -> after a few seconds (if the health check is immediate), or "within 30 seconds" if the health check might not occur until 30 sec later. (I don't remember which health checks got changed to immediate).

Reply:
ATM it's not until about 30 seconds later, so I'll change it to "within 30 seconds" and we can change that section later once that's implemented